.github/dependabot.yml

```diff
@@ -0,0 +1,22 @@
+# To get started with Dependabot version updates, you'll need to specify which
+# package ecosystems to update and where the package manifests are located.
+# Please see the documentation for all configuration options:
+# https://help.github.com/github/administering-a-repository/configuration-options-for-dependency-updates
+
+version: 2
+updates:
+  - package-ecosystem: "npm"
+    directory: "/"
+    schedule:
+      interval: "daily"
+    ignore:
+      - dependency-name: "govuk-frontend"
```

Comment:
I think @solidgoldpig wanted us to do the same for `moj-frontend`.

And there are [two packages](https://github.com/ministryofjustice/hmpps-book-secure-move-frontend/blob/7bfa5edb1af9f2b02cbe3ffcdc4b6e91ec9d9027/.ncurc.json#L2) that cause failures that we might also want to ignore for now:
- `css-loader`
- `webpack-merge`

Reply:
We'll have to ignore them for now if we don't want the app to be broken.

https://github.com/ministryofjustice/hmpps-book-secure-move-frontend/issues/712

Reply:
Yep, that was my suggestion for now. We add them to the ignore list as well.

Reply:
I've set the `css-loader` and `webpack-merge` to be below the next version that's a breaking change. This should mean that any patch updates will come through.

`moj-frontend` and `govuk-frontend` are both set to completely ignore.

Reply:
Also added `@hmpps-book-secure-move-frameworks*` to the list, as that is also an intentional upgrade we'd need to perform